CarpetX/src/prolongate_3d_rf2_impl.hxx

```diff
@@ -187,33 +187,39 @@ template <typename T> struct coeffs1d<CC, POLY, /*order*/ 11, T> {
 
 // Hermite interpolation (with matched first derivatives)
 
-// Linear Hermite interpolation is the same as linear Lagrange interpolation
-template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 1, T> {
-  static constexpr std::array<T, 4> coeffs = {
-      +1 / T(2),
-      +1 / T(2),
-  };
-};
-// Cubic Hermite interpolation is the same as cubic Lagrange interpolation
 template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 3, T> {
-  static constexpr std::array<T, 4> coeffs = {
-      -1 / T(16),
-      +9 / T(16),
-      +9 / T(16),
-      -1 / T(16),
+  static constexpr std::array<T, 6> coeffs = {
+      +1 / T(96),
+      -9 / T(96),
+      56 / T(96),
+      56 / T(96),
+      -9 / T(96),
+      +1 / T(96)
```

Comment:
while I agree that Hermite of same order needs more points, I suspect that we may need to (same as Carpet) read "order" really as "this many ghost points" to be able to mix operators of different orders using the same number of ghost points.